src/index.scss

```diff
@@ -0,0 +1,12 @@
+body {
```

Comment:
this contentt should be on main `application.scss` with all the needed global sass imports